packages/SwingSet/test/test-state.js

```diff
@@ -697,9 +697,9 @@ test('crankhash - skip keys', t => {
   // certain local keys are excluded from consensus, and should not affect
   // the hash
   k.kvStore.set('one', '1');
-  k.kvStore.set('local.snapshot.XYZ', '["vat1234"]');
+  k.kvStore.set('local.xsnapshot.XYZ', '["vat1234"]');
   k.kvStore.set(
-    'local.v1234.lastSnapshot',
+    'local.v1234.xlastSnapshot',
```

Comment:
These two keys are no longer representative of things that get stored in a real kvStore, right? Maybe we should replace them with e.g. `local.doNotHashMe` instead of referencing an obsolete key string.